usaspending_api/common/serializers.py

```diff
@@ -7,83 +7,127 @@ class LimitableSerializer(serializers.ModelSerializer):
 
     """Extends the model serializer to support field limiting."""
     def __init__(self, *args, **kwargs):
-        # next two lines are deprecated and will be removed
-        # once all views inherit from a generic class or are
-        # refactored into viewsets (i.e., once the serializer
-        # consistently has direct access to the request)
-        include_fields = kwargs.pop('fields', None)
-        exclude_fields = kwargs.pop('exclude', None)
+        # Grab any kwargs include and exclude fields, these are typically
+        # passed in by a parent serializer to the child serializer
+        kwargs_has_include = 'fields' in kwargs
```

Comment:
Just to make sure I understand what's happening here... by checking `**kwargs` for include and exclude fields, the views/viewsets to use the `get_serializer_context()` method to override the field defaults as necessary?

If that's true, do we need to do that in whatever viewset (tas/file a something or another?) returns the non-default list of fields? Or am I misunderstanding (entirely possible)?